src/main/java/gov/usgs/owi/nldi/controllers/LookupController.java

```diff
@@ -24,6 +24,9 @@
 @RestController
 public class LookupController extends BaseController {
 
+	//swagger documentation for /lookups/{characteristicType}/characteristics endpoint
+	@Operation(summary = "getCharacteristics", description = "Returns available characteristics metadata")
```

Comment:
You need to move this annotation below the constructor, to the method it modifies ... around line 33.

And also this file needs an import:


import io.swagger.v3.oas.annotations.Operation;